README.md

```diff
@@ -25,51 +25,68 @@ Next up, the service provider must be registered:
 ```php
 'providers' => [
     ...
-    Spatie\FailedJobMonitor\FailedJobMonitorServiceProvider::class,
+    Spatie\FailedJobMonitor\ServiceProvider::class,
     ...
 ];
 ```
 
 Next, you must publish the config file:
 
 ```bash
-php artisan vendor:publish --provider="Spatie\FailedJobMonitor\FailedJobMonitorServiceProvider"
+php artisan vendor:publish --provider="Spatie\FailedJobMonitor\ServiceProvider"
 ```
 
-This is the contents of the configuration file. Most options are self-explanatory.
+This is the contents of the default configuration file. 
 
 ```php
 return [
+    '*' => [
```

Comment:
I'd like to keep this package very simple. Adding job-specific config brings a lot of complexity with it. My thinking is that most users will want a notification for every job that fails.  Could you remove this functionality throughout this PR?